### ISSUE TITLE
Ambiguous wording: Split or Split

### ISSUE THREAD

**Issue by Shah-Nisarg** [dotnet/docs#7362]
In the first section, one of the paragraph states:

> LINQ queries can be combined with traditional string functions and regular expressions. For example, you can use the [Split](https://docs.microsoft.com/en-us/dotnet/api/system.string.split) or [Split](https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regex.split) method to create an array of strings that you can then query or modify by using LINQ.

Notice that while reading it is unclear which `Split` methods are being referenced. Hovering on the links does reveal the class-namespaces being referenced.

I would suggest rewording the sentence as following to improve readability:

> LINQ queries can be combined with traditional string functions and regular expressions. For example, you can use the [String.Split](https://docs.microsoft.com/en-us/dotnet/api/system.string.split) or [Regex.Split](https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regex.split) method to create an array of strings that you can then query or modify by using LINQ.

There is a similar instance in section "Querying a Block of Text". So that one would also require rewording.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b2730d0d-f5e5-9948-5566-008fb42b8a2d
* Version Independent ID: a9b9f250-0107-cf1b-c182-469c7fbe5fb5
* Content: [LINQ and Strings (C#)](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/linq-and-strings)
* Content Source: [docs/csharp/programming-guide/concepts/linq/linq-and-strings.md](https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/concepts/linq/linq-and-strings.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by mairaw** [comment]
Thanks for your feedback @Shah-Nisarg. I've opened a PR to fix this:
https://github.com/dotnet/docs/pull/7374
